src/api/entities/Identity/__tests__/index.ts

```diff
@@ -1205,23 +1230,27 @@ describe('Identity class', () => {
 
   describe('method: getSecondaryAccounts', () => {
     const accountId = 'someAccountId';
-    const signerAccountId = dsMockUtils.createMockSignatory({
-      Account: dsMockUtils.createMockAccountId(accountId),
-    });
 
     let account: Account;
     let fakeResult: PermissionedAccount[];
 
-    let signatoryToAccountStub: sinon.SinonStub<[Signatory, Context], Account>;
-    let didRecordsStub: sinon.SinonStub;
-    let rawDidRecord: PolymeshPrimitivesIdentity;
+    let rawPrimaryKeyRecord: PolymeshPrimitivesSecondaryKeyKeyRecord;
+    let rawSecondaryKeyRecord: PolymeshPrimitivesSecondaryKeyKeyRecord;
+    let rawMultiSigKeyRecord: PolymeshPrimitivesSecondaryKeyKeyRecord;
+    let rawDidRecord: { args: [PolymeshPrimitivesIdentityId, string] };
```

Comment:
I saw there was a pattern for this with tuple. Will look into it more in the morning